README.md

```diff
@@ -180,7 +245,7 @@ this README. However, a few details about how the job is configurd are provided
 * The `env.json` file described in the **Configuration** section above needs to be made available to 
   running course-inventory containers via an OpenShift ConfigMap, a type of Resource. A volume containing the ConfigMap 
   should be mapped to the `config/secrets` subdirectory. These details will be specified in a configuration file
-  (.yaml) defining the pod.
```

Comment:
This currently does have .yaml as the extension. Maybe just remove it entirely or don't use an extension as it could be either .yaml or .yml.

FYI: [Is there an official extension for YAML files? Please use ".yaml" when possible.
](https://web.archive.org/web/20060924190202/https://yaml.org/faq.html) We aren't limited by 8.3 anymore. ;) This advice is unchanged since 2006.

Reply:
The current version has some references to `yaml` and some to `yml`.  Some of those are as filename extensions, and some are not.  I'll just update all the references to "YAML".